src/lib.rs

```diff
@@ -320,6 +327,36 @@ pub async fn compare_attestations(
     let mut remote_attestations = remote_attestations.clone();
     remote_attestations.sort_by(|a, b| a.stake_weight.partial_cmp(&b.stake_weight).unwrap());
 
+    if !remote_attestations.is_empty() {
```

Comment:
Would it make sense to include empty remote_attestations in the metrics with a default 0 if we ever have such an entry? In theory we don't create empty vec for remote_attestations and the case would get into L320, right?

Reply:
yep! I guess the check is redundant